src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -494,6 +557,20 @@ def build(self, notification_uuid: str | None = None) -> Union[SlackBlock, Slack
             action_block = {"type": "actions", "elements": [action for action in actions]}
             blocks.append(action_block)
 
+        # suggested assignees
+        suggested_assignees = []
+        if event_for_tags:
+            suggested_assignees = get_suggested_assignees(
+                self.group.project, event_for_tags, assignee
+            )
+        if len(suggested_assignees) > 0:
+            suggested_assignee_text = "Suggested Assignees: "
+            for idx, assignee in enumerate(suggested_assignees):
+                if idx != 0:
```

Comment:
nit: I believe `enumerate` can optionally take a `start` position, so you can just start at 1 and not have to check this